### PR TITLE
Add `dactyl_tracer` board

### PR DESCRIPTION
**Pull request by dunk2k**

### CODE REVIEW

File: boards/dactyl_tracer/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Dactyl Tracer
+
+![TopElev](https://i.imgur.com/ReCBppEh.jpeg)
+
+A split body, concave shaped key well, columnar keyboard, with case halves designed similarly to the [Dactyl CC](/boards/dactyl/README.md#dactyl-cc--ergo-s-1).  
+The 'Q' key is vertically offset, from 'A' key, to allow more comfortable actuation using ring finger.
```
Comment (xs5871) [inline boards/dactyl_tracer/README.md:6]: Allthough I'm a big fan of single quotes in code, I argue it should be double quotes in plain text for "Q" and "A". Did you find any established format preference in the docs?

File: boards/dactyl_tracer/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Dactyl Tracer
+
+![TopElev](https://i.imgur.com/ReCBppEh.jpeg)
+
+A split body, concave shaped key well, columnar keyboard, with case halves designed similarly to the [Dactyl CC](/boards/dactyl/README.md#dactyl-cc--ergo-s-1).  
+The 'Q' key is vertically offset, from 'A' key, to allow more comfortable actuation using ring finger.  
+Key arrangement bears slight resemblance to the [Dactyl ManuForm](/boards/dactyl_manuform/) in that:
+1. Row spacing and placement of two outward most thumb keys are identical
+2. Bottom finger key-well row has two keys
```
Comment (xs5871) [inline boards/dactyl_tracer/README.md:9]:
```suggestion
1. Row spacing and placement of two outward most thumb keys are identical and
2. bottom finger key-well row has two keys.
```
That's a suggestion, the exact wording and punctuation I leave up to you -- but: if a list reads like a sentence, it should adhere to the syntax of a sentence.

File: boards/dactyl_tracer/README.md (hunk 1 above)
Comment (dunk2k) [reply boards/dactyl_tracer/README.md:6]: Nothing [here](​​http://kmkfw.io/docs/contributing#contributing-documentation) about quote preference for documentation.

Comment (dunk2k) [reply boards/dactyl_tracer/README.md]: single quotes changed to double quotes.

**Review by xs5871 — APPROVED** [review]

**Pull request merged**